docs/documentation/updating-the-kit.md

```diff
@@ -55,6 +55,14 @@ It will download a zip file and unzip the latest version of the Prototype Kit in
 
 11. Delete the update folder in Finder or Windows Explorer.
 
+12. If your prototype is already on GitHub, make sure that users with a local copy will get any file changes you make. To update this kind of prototype:
+
+  - [branch off the `main` (or default) branch](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/managing-branches)
```

Comment:
I included this link (and the ones below) in case it might help users who aren't that familiar with GitHub. But let me know if we should remove them for any reason!